src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier supplier = new ColorSupplier();
+        ball.setColor(supplier.getRandomColor());
+        Random random = new Random();
+        ball.setNumber(random.nextInt(100));

```

Comment:
magic number

Reply:
what do you mean?

Reply:
[All magic numbers in your code should be constants.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants)